ballerina/tests/xml_to_json_test.bal

```diff
@@ -353,9 +353,9 @@ isolated function testFromXMLWithXmlSequence() {
 }
 
 public function convertToJson(string xmlStr) returns string = @java:Method {
-    'class: "org/ballerinalang/stdlib/xmldata/testutils/XmlDataTestUtils"
+    'class: "io/ballerina/stdlib/xmldata/testutils/XmlDataTestUtils"
```

Comment:
Even though the path separator is working, the recommended way is to use the dot separator. 
Check other places as well. 

```suggestion
    'class: "io.ballerina.stdlib.xmldata.testutils.XmlDataTestUtils"
```